modules/assimp/editor_scene_importer_assimp.cpp

```diff
@@ -146,11 +136,11 @@ Node *EditorSceneImporterAssimp::import_scene(const String &p_path, uint32_t p_f
 								 //aiProcess_OptimizeGraph |
 								 //aiProcess_Debone |
 								 aiProcess_EmbedTextures |
```

Comment:
Didn't we say embed texture caused problems? I forget.

Reply:
Removed! :)